mage_ai/services/spark/spark.py

```diff
@@ -2,6 +2,33 @@
 import os
 
 
+def get_list_of_file_names(list_of_jars):
+    if list_of_jars is None:
+        return None
+
+    return [os.path.basename(jar) for jar in list_of_jars]
+
+
+def contains_same_jars(list_of_jars_1, list_of_jars_2):
+    if list_of_jars_1 is None and list_of_jars_2 is None:
+        return True
+
+    if list_of_jars_1 is None or list_of_jars_2 is None:
+        return False
+
+    if len(list_of_jars_1) != len(list_of_jars_2):
+        return False
+
+    file_names_1 = get_list_of_file_names(list_of_jars_1)
+    file_names_2 = get_list_of_file_names(list_of_jars_2)
+
+    for file_name in file_names_1:
```

Comment:
You can compare them by converting them to set

Reply:
Made the suggested change.